upgrade/helpers/constants.py

```diff
@@ -2,7 +2,7 @@
 
 rhelcontents = {
     'rhscl': {
-        'prod': 'Red Hat Software Collections (for RHEL Server)',
+        'prod': 'Red Hat Software Collections for RHEL Server',
```

Comment:
Any specific reason to change this one ? Was this working fine till last release ?

Reply:
the product name has changed.